tests/test-backend-ops.cpp

```diff
@@ -763,7 +763,8 @@ struct test_case {
 
         ggml_tensor * out = build_graph(ctx.get());
 
-        if ((op_name != nullptr && op_desc(out) != op_name) || out->op == GGML_OP_OPT_STEP_ADAMW) {
+        if ((op_name != nullptr && op_desc(out) != op_name) || out->op == GGML_OP_OPT_STEP_ADAMW ||
+            out->op == GGML_OP_OPT_STEP_SGD) {
```

Comment:
```suggestion
        if ((op_name != nullptr && op_desc(out) != op_name) || out->op == GGML_OP_OPT_STEP_ADAMW ||
                out->op == GGML_OP_OPT_STEP_SGD) {
```

The identation should either be visually distinct from the line below or there should be a blank line separating them.

Reply:
ok